cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -902,6 +957,13 @@ pub enum LockedManifestError {
 
     #[error("Catalog lockfile does not support update")]
     UnsupportedLockfileForUpdate,
+
+    #[error("The package '{0}' has license '{1}' which is not in the list of allowed licenses.\n\nAllow this license by adding it to 'options.allow.licenses' in manifest.toml")]
+    LicenseNotAllowed(String, String),
+    #[error("The package '{0}' is marked as broken.\n\nAllow broken packages by setting 'options.allow.broken = true' in manifest.toml")]
+    BrokenNotAllowed(String),
+    #[error("The package '{0}' has an unfree license.\n\nAllow unfree packages by setting 'options.allow.unfree = true' in manifest.toml")]
+    UnfreeNotAllowed(String),
```

Comment:
nit: i'd implement this formatting in flox/utils/errors.rs but fwiw its reasonably distinct from the cli to just stay here

Reply:
I find it more readable if the string on an error is actually what gets printed. I think it's more confusing if that string is discarded or modified somewhere else. But we should settle on one approach at some point